build.gradle

```diff
@@ -116,6 +116,7 @@ dependencies {
     compile group: 'com.jayway.jsonpath', name: 'json-path', version: '2.1.0'
     compile group: 'com.fasterxml.jackson.core', name:'jackson-databind', version: '2.6.4'
     compile group: 'com.fasterxml.jackson.datatype', name:'jackson-datatype-jdk8', version: '2.6.4'
+    compile group: 'com.fasterxml.jackson.datatype', name:'jackson-datatype-guava', version: '2.6.4'
```

Comment:
Might want to extract the version number since it's used multiple times related to jackson